.github/workflows/nightly.yml

```diff
@@ -0,0 +1,35 @@
+name: Nightly Test
+
+on:
+  schedule:
+    - cron: '15 3 * * *'
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+  CONTAINER_RUNTIME: 'docker'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image_tag:
+          - "latest"
+          - "3.8"
+          - "3.7"
+          - "3.6"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
Not sure, it is a good idea, to use a python version not covered by the PR workflows.